src/algebra/ring/aut.lean

```diff
@@ -60,4 +61,26 @@ by refine_struct { to_fun := ring_equiv.to_mul_equiv }; intros; refl
 def to_perm : ring_aut R →* equiv.perm R :=
 by refine_struct { to_fun := ring_equiv.to_equiv }; intros; refl
 
+/-- The tautological action by the group of automorphism of a ring `R` on `R`.-/
+instance apply_mul_semiring_action {R : Type*} [ring R] : mul_semiring_action (ring_aut R) R :=
```

Comment:
I expect this holds for weaker assumptions than `ring R`. My guess would be you want just
```suggestion
instance apply_mul_semiring_action : mul_semiring_action (ring_aut R) R :=
```
that is, the file already has the right assumptions.

Reply:
Hi Eric, thanks for the suggestion - locally I get errors if I remove these assumptions but I pushed it anyways to see if it works.

Reply:
Yeah, it doesn't work. Which assumptions need to change on the file?

Reply:
Ah, thanks for trying. In that case
```suggestion
instance apply_mul_semiring_action {R : Type*} [semiring R] : mul_semiring_action (ring_aut R) R :=
```

Reply:
(and below)